src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -86,6 +87,7 @@
     public static final int MAX_RECORDS_IN_RAM = 500000;
 
     private static final char[] RESERVED_ATTRIBUTE_STARTS = {'X', 'Y', 'Z'};
+    private static final int MAX_FOREIGN_CONTAM_WARN = 1_000;
```

Comment:
since the output in the tag is `cross-species` perhaps we change `foreign` to `cross-species` throughout?